### PR TITLE
small tweaks to archiving instructions

### PR DESCRIPTION
**Pull request by simonpcouch**
Based on #379 and #291, a follow-up to #196 with some updates reflecting the reality of how this has actually played out in practice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ execute:
 ```
 * In the command line, run `quarto render archive/YYYY-MM-workshop-name`. This will regenerate the workshop slides under `docs/archive/YYYY-MM-workshop-name`.
 * Check that:
-	* Running `quarto render` didn't change any files in `docs/` outside of `docs/archive/`.
+	* Running `quarto render` didn't change any files in `docs/` other than `sitemap.xml` and `search.json` outside of `docs/archive/`.
```
Comment (hfrick) [inline README.md:39]: This should mention `index.html` as the third file allowed to change inside `docs/archive/`, no?

**Review by hfrick — APPROVED** [review]
Thanks for improving the instructions!

File: README.md (hunk 1 above)
Comment (simonpcouch) [inline README.md:39]:
```suggestion
	* Running `quarto render` didn't change any files in `docs/` other than `index.html`, `sitemap.xml`, and `search.json` outside of `docs/archive/`.
```

Comment (simonpcouch) [reply README.md]: Right, yes!

**Pull request merged**